hooks/cspell/turo-dictionary.txt

```diff
@@ -0,0 +1,133 @@
+# This is a comment
+# "Forbidden" syntax: !forbiddenword
+allstar
```

Comment:
This is the dictionary where we can add terms we run into in stand-alone repos to be used across all repos. Anything Turo-specific or Engineering-specific or general terminology can go here. 

Any terms that are more likely to be repo-specific can go in an array maintained in the repo itself: https://github.com/turo/web-schumacher-app/pull/9234/files#diff-303b67a39635355281d1863a42d6648fbaab26cc0ea8c4c5f3bff37eea65cabb